addons/im_livechat/models/res_partner.py

```diff
@@ -55,6 +58,20 @@ def _compute_user_livechat_username(self):
         for partner in self:
             partner.user_livechat_username = next(iter(partner.user_ids.mapped('livechat_username')), False)
 
+    def _compute_livechat_channel_count(self):
+        channel_count_dict = dict(
+            self.env["discuss.channel.member"]._read_group(
```

Comment:
Actually, it's still not relying on member history. Members can leave the conversation so this is not correct.